Sources/Public/CalendarView.swift

```diff
@@ -652,16 +652,13 @@ public final class CalendarView: UIView {
   private func configureView(_ view: ItemView, with visibleItem: VisibleCalendarItem) {
     view.calendarItemModel = visibleItem.calendarItemModel
 
-    // Update the visibility
-    UIView.performWithoutAnimation {
```

Comment:
previously, we were preventing animations for frame updates. We only need to prevent animations if an item is being reused. There are legit use cases where the frame might update (and we'd want to animate), like changing the day aspect ratio, or the spacing between elements.